### PR TITLE
Remove Services Rigger from Kubernetes upgrade

### PR DESCRIPTION
**Pull request by laurentdroin**
Unless I misunderstand something, the services rigger is upgraded with the REC, not when applying the bundle.

### CODE REVIEW

File: content/platforms/kubernetes/upgrading-kubernetes-with-operator.md [hunk 1]
```diff
@@ -75,4 +75,4 @@ Each pod is updated after the last one completes successfully.
 
 The pods in the StatefulSet are updated in reverse ordinal order.
 The Kubernetes controller terminates each pod and waits for it to transition to `Running` and then to `Ready`.
-The it updates the next pod until all pods are updated and the upgrade process is completed.
\ No newline at end of file
+The it updates the next pod until all pods are updated and the upgrade process is completed.
```
Comment (bmansheim) [inline content/platforms/kubernetes/upgrading-kubernetes-with-operator.md:78]: What's "The it"?

Comment (laurentdroin) [reply content/platforms/kubernetes/upgrading-kubernetes-with-operator.md:78]: Hmmm... I don't remember adding that line. I just wanted to remove the reference to the Services Rigger (line 38). Let me think about it some more.

Comment (laurentdroin) [reply content/platforms/kubernetes/upgrading-kubernetes-with-operator.md:78]: This is strange. I don't think that line is from me. This is not my style. I don't think I would use the expression "update a pod". Still checking.

Comment (laurentdroin) [reply content/platforms/kubernetes/upgrading-kubernetes-with-operator.md:78]: The change was already there in PR #749

Comment (alexmilowski) [reply content/platforms/kubernetes/upgrading-kubernetes-with-operator.md:78]: I think this is out of date now so the PR needs to be updated. We have confirmed this is the correct behaviour and so removing "Service Rigger" from the step is technically correct.

Comment (alexmilowski) [reply content/platforms/kubernetes/upgrading-kubernetes-with-operator.md:78]: Must be a line-ending change on that last line. I think we should change that line to:

"Once a pod is updated, the next pod is terminated and updated. Once all the pods are updated, the upgrade process is completed."

Comment (bmansheim) [reply content/platforms/kubernetes/upgrading-kubernetes-with-operator.md]: Done

**Pull request merged**